cypress/support/commands.js

```diff
@@ -216,3 +216,20 @@ Cypress.Commands.add('createResponseTemplate', ({ title, content }) => {
     `utf8=%E2%9C%93&response_template%5Btitle%5D=${encodedTitle}&response_template%5Bcontent%5D=${encodedContent}`,
   );
 });
+
+/**
+ * Creates a listing.
+ *
+ * @param {string} title The title of a a listing.
+ * @param {string} [content=''] The content of the listing.
+ *
+ * @returns {Cypress.Chainable<Cypress.Response>} A cypress request for creating a listing.
+ */
+Cypress.Commands.add('createListing', ({ title, content }) => {
```

Comment:
this is great to have 🙌

Reply:
I just realized it doesn't work though :( I can't get it to work from JS, I'm going to delete the listing created in the seed file